app/views/admin/events/edit.html.erb

```diff
@@ -31,9 +31,10 @@
   <p><%= f.label(:confirmation_date, "Attendance to be confirmed by:") %></p>
   <%= error_message(@event, :confirmation_date) %>
   <p><%= f.date_field :confirmation_date %></p>
-
 </fieldset>
+
 <div class='button'>
   <%= f.submit "Save", class: 'commit', type: 'submit' %>
+  <%= button_to "Delete event", { action: "destroy", id: @event.id }, method: :delete, data: { confirm: "Are you sure?" } %>
```

Comment:
The `action:` option is not needed, the `destroy` method will be executed mecause the HTTP method is `delete`: http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

I don't think that's the reason why the tests don't pass, we can have a look in today's meetup.